project-fortis-interfaces/src/components/Admin/Streams/CreateStream.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import Form from 'react-jsonschema-form';
+import { Card, CardTitle, CardText } from 'material-ui/Card';
+import Divider from 'material-ui/Divider';
+import StreamConstants from './StreamConstants';
+import { guid } from './../../../utils/Utils';
+import forOwn from '../../../../node_modules/lodash/forOwn';
+
+class CreateStream extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.save = this.save.bind(this);

```

Comment:
Given that you're declaring save as a class-method with an arrow function below, you don't need bind. Check out [this article](https://medium.freecodecamp.org/react-binding-patterns-5-approaches-for-handling-this-92c651b5af56) for a great overview of the various approaches to preserving the this-context.

In short, either you do this:

```js
constructor() {
  this.foo = this.foo.bind(this);
}

foo() {
  // ...
}
```

Or you do just this:

```js
foo = () => {
  // ...
}
```